### PR TITLE
compatibility with doctrine/dbal 3.1

### PR DESCRIPTION
**Pull request by othillo**
fixes #46

### CODE REVIEW

File: src/DBALEventStore.php [hunk 1]
```diff
@@ -293,27 +295,25 @@ private function convertStorageValueToIdentifier($id)
 
     public function visitEvents(Criteria $criteria, EventVisitor $eventVisitor): void
     {
-        $statement = $this->prepareVisitEventsStatement($criteria);
-        $statement->execute();
+        /** @var Result $result */
```
Comment (wjzijderveld) [inline src/DBALEventStore.php]: nit: method has a proper return type, so this does nothing now, right?

File: test/BinaryDBALEventStoreTest.php [hunk 2]
```diff
@@ -61,7 +56,8 @@ public function table_should_contain_binary_uuid_column()
         $uuidColumn = $this->table->getColumn('uuid');
 
         $this->assertEquals(16, $uuidColumn->getLength());
-        $this->assertEquals(Type::getType(Type::BINARY), $uuidColumn->getType());
+
+        $this->assertEquals(Type::getType('binary'), $uuidColumn->getType());
```
Comment (wjzijderveld) [inline test/BinaryDBALEventStoreTest.php]: Constants have moved to `Types`, so you can use `Types::BINARY`

**Review by wjzijderveld — APPROVED** [review]
2 minor comments, but looks good to me :+1:

File: test/BinaryDBALEventStoreTest.php (hunk 2 above)
Comment (othillo) [reply test/BinaryDBALEventStoreTest.php]: Thanks! This felt wrong indeed :)

**Pull request merged**